### PR TITLE
Add Panquesito7 to the heptagram-devs

### PR DESCRIPTION
**Pull request by MidouWebDev**
I added panquesito7 😉

### CODE REVIEW

**Comment by MidouWebDev** [comment]
@j-dogcoder

**Comment by j-dogcoder** [comment]
Making my way through PRs now.

**Review by j-dogcoder — APPROVED** [review]

File: heptagram-devs.html [hunk 1]
```diff
@@ -40,9 +40,15 @@ <h1>Developers</h1>
         <div class="head-developer">
             <img class="avatar" src="img/midou's avatar.png" alt="midou's avatar">
```
Comment (Panquesito7) [inline heptagram-devs.html:41]: One question: can't we take the images from GitHub instead of saving them locally?

Comment (j-dogcoder) [reply heptagram-devs.html:41]: We actually should do that. @MidouWebDev.... Want to do this? Or have @Panquesito7 work on it?

Comment (Panquesito7) [reply heptagram-devs.html:41]: > We actually should do that. @MidouWebDev.... Want to do this? Or have @Panquesito7 work on it?

I would like to work on it, but if @MidouWebDev wants to, feel free to work on that. 🙂

Comment (j-dogcoder) [reply heptagram-devs.html:41]: It is up to him

Comment (MidouWebDev) [reply heptagram-devs.html:41]: @Panquesito7 you can do it, bro !

Comment (Panquesito7) [reply heptagram-devs.html:41]: Thanks! Working on it. 😄

**Pull request merged**